showpoll.php

```diff
@@ -119,199 +99,143 @@
 <head>
   <title>admin dashboard</title>
   <link rel="stylesheet" type="text/css" href="style.css">
-  
 </head>
 <body>
-<a href="poll.php">poll again</a>
-
-    <div class="admin">
-
-  <?php
+<a href="poll.php">Add New Poll</a>
+<div class="admin">
+<?php
  $showRecordPerPage = 5;
- 
-
  if (isset($_GET['page']) && !empty($_GET['page'])) {
      $currentPage = $_GET['page'];
-     
-    
- }else{
+  }
+  else{
    $currentPage = 1;
- }
-      $startFrom = ($currentPage * $showRecordPerPage) - $showRecordPerPage;
-      $query="select * from polls";
-      $run = mysqli_query($db,$query);
-      $totalpoll = mysqli_num_rows($run);
-     
-      $pages = $totalpoll;
-      
-     
-   
-      $lastPage = ceil($totalpoll/$showRecordPerPage);
-      $firstPage = 1;
-      $nextPage = $currentPage + 1;
-      $previousPage = $currentPage - 1;
-     
-
- $empSQL ="SELECT `id`, `Question`, `Options`, `Is_active`, `created time`, `last edit` FROM `polls` LIMIT $startFrom, $showRecordPerPage";
- $run = mysqli_query($db, $empSQL);
-
-
-
-
-
-//
+  }
+$startFrom = ($currentPage * $showRecordPerPage) - $showRecordPerPage;
+$query="select * from polls";
+$run = mysqli_query($db,$query);
+$totalpoll = mysqli_num_rows($run);
+$pages = $totalpoll;
+$lastPage = ceil($totalpoll/$showRecordPerPage);
+$firstPage = 1;
+$nextPage = $currentPage + 1;
+$previousPage = $currentPage - 1;
+$empSQL ="SELECT `id`, `Question`, `Options`, `Is_active`, `created time`, `last edit` FROM `polls` LIMIT $startFrom, $showRecordPerPage";
+$run = mysqli_query($db, $empSQL);
 $isEditMode = false;
 $isDeleteMode = false;
-if( isset($_GET['edit_id']) ){
-    
+if (isset($_GET['edit_id'])) {
     $isEditMode = $_GET['edit_id'];
- 
 }
-//
- 
-
-
-//
-   
-
-   if($run==true)
-   {
+if($run==true){
+?>
+<table border="1">
+<tr bgcolor="pink">
+<td>Id</td>
+<td>question</td>
+<td colspan="4">option</td>
+<td>Edit</td>
+<td>Delete</td>
+<td>status</td>
+<td>created time</td>
+<td>lastedit time</td>
+</tr>
+<?php
+while($data=mysqli_fetch_assoc($run)){ 
+  $option= $data['Options'];
+  $value=json_decode($option);
+    if( $isEditMode != $data['id']){
+?>
+      <form method="post">
+      <tr>
+      <td><?php echo $data['id']; ?><br/><br/></td>
+      <td><?php echo $data['Question']; ?><br/><br/></td>
+      <?php
+      for($i=0;$i<=3;$i++){?>
```

Comment:
don't use static length for array iteration. use count function for get array count.
Also use foreach loop.